internal/authz/authz.go

```diff
@@ -169,6 +191,65 @@ func (a *Authorizer) AuthorizeCreateStore(ctx context.Context) error {
 	return a.individualAuthorize(ctx, claims.ClientID, relation, System, &openfgav1.ContextualTupleKeys{})
 }
 
+// GetModulesForWriteRequest returns the modules that should be checked for the write request.
+// If we encounter a type with no attached module, we should break and return no modules so that the authz check will be against the store
```

Comment:
Please help me understand this comment. If i have

```
type user

// module A
type doc

// module B
type group
```

and I try to write three tuples
- doc:1, viewer, user:maria
- group:2, viewer, user:maria
- group:2, parent, group:3

the Write as whole will go through because `user` has no module linked to it? But i may not have permissions to write that last tuple 🤔